### PR TITLE
fix(platform): multi input issues 

### PR DESCRIPTION
**Pull request by genereg1**
## Related Issue(s)

part of https://github.com/SAP/fundamental-ngx/issues/7606

## Description

- Remove `Decline example` and support`[selectionMode]='delete'` Deleting tokens from the list should work by usual click on item. Instead created an example with More Token label that shows the list with selected tokens and [X] button
- Small improvements 

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [x] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [x] API boundary values
-   [x] different combinations of components - free style
-   [x] change the API values during testing

3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable
-   [x] poor examples
-   [x] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [ x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [x] Run npm run build-pack-library and test in external application
-   [n/a] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

**Comment by mikerodonnell89** [comment]
@genereg1 is this still considered a "draft" pull request or shall we start reviewing it?

**Comment by genereg1** [comment]
> @genereg1 is this still considered a "draft" pull request or shall we start reviewing it?

@mikerodonnell89 still on draft. Working on broken e2e tests

File: libs/platform/src/lib/form/multi-input/base-multi-input.ts [hunk 1]
```diff
@@ -218,44 +224,56 @@ export abstract class BaseMultiInput extends CollectionBaseInput implements Afte
...
      * @hidden
+     * Need for opening mobile version
      */
     openChange = new Subject<boolean>();
 
+    /**
+     * @hidden
+     * Need for opening more label selected items list
+     */
+    showSelectedList$ = new BehaviorSubject<any>(false);
```
Comment (platon-rov) [inline libs/platform/src/lib/form/multi-input/base-multi-input.ts:261]: `BehaviorSubject<boolean>`

File: libs/platform/src/lib/form/multi-input/base-multi-input.ts [hunk 2]
```diff
@@ -692,11 +730,15 @@ export abstract class BaseMultiInput extends CollectionBaseInput implements Afte
     private _convertObjectsToDefaultOptionItems(items: any[]): MultiInputOption[] {
         const selectItems: MultiInputOption[] = [];
 
+        console.log('select', this._selectedItems);
+        console.log('value', items);
```
Comment (platon-rov) [inline libs/platform/src/lib/form/multi-input/base-multi-input.ts:734]: please remove.

File: libs/platform/src/lib/form/multi-input/multi-input.component.ts [hunk 3]
```diff
@@ -78,21 +80,20 @@ export class PlatformMultiInputComponent extends BaseMultiInput implements OnIni
     listTemplateDD: ListComponent<MultiInputOption>;
 
     /** Selected values from the list items. */
-    _selected: any[] = [];
-
     @Input()
-    set selected(selectedValue: any[]) {
+    set selectedItems(selectedValue: any[]) {
```
Comment (platon-rov) [inline libs/platform/src/lib/form/multi-input/multi-input.component.ts:84]: breaking change.

**Pull request closed without merging**